src/core/DynaRec_x64/recompiler.h

```diff
@@ -181,6 +191,12 @@ class DynaRecCPU final : public PCSX::R3000Acpu {
         }
     }
 
+    virtual void invalidateCache() final {
+        memset(m_psxRegs.ICache_Addr, 0xff, sizeof(m_psxRegs.ICache_Addr));
+        memset(m_psxRegs.ICache_Code, 0xff, sizeof(m_psxRegs.ICache_Code));
+        m_invalidateBlocks();
```

Comment:
We may want to also call base, to clear the interpreter's cache.

Reply:
The 2 memsets should clean the interp cache though?

Reply:
Right, that was kind of my point. It's probably better to call the base `R3000Acpu::invalidateCache()` instead of duplicating the memset code, in case the base function changes later on.